code/game/jobs/squads.dm

```diff
@@ -72,11 +72,31 @@
 	name = "Delta"
 	id = DELTA_SQUAD
 	tracking_id = TRACK_DELTA_SQUAD
-	color = 4
+	color = "#4148c8" // rgb(65,72,200)
 	access = list(ACCESS_MARINE_DELTA)
 	usable = TRUE
 	radio_freq = DELTA_FREQ
 
+GLOBAL_LIST_EMPTY(armormarkings)
+GLOBAL_LIST_EMPTY(armormarkings_sql)
+GLOBAL_LIST_EMPTY(helmetmarkings)
+GLOBAL_LIST_EMPTY(helmetmarkings_sql)
+
+/datum/squad/New()
+	. = ..()
+	var/image/armor = image('icons/mob/suit_1.dmi',icon_state = "std-armor")
+	var/image/armorsl = image('icons/mob/suit_1.dmi',icon_state = "sql-armor")
+	armor.color = color
+	armorsl.color = color
+	GLOB.armormarkings[type] = armor
+	GLOB.armormarkings_sql[type] = armorsl
```

Comment:
Could you use `armormarkings_sl` instead of `armormarkings_sql` ? Squad leader is not the first thing that comes to mind by SQL.